Gemfile.lock

```diff
@@ -444,4 +446,4 @@ DEPENDENCIES
   zeus
 
 BUNDLED WITH
-   1.14.6
+   1.16.0
```

Comment:
This will create some havoc on the infrastructure side if I'm not mistaken. Also the sqlie3 dep must be added to the omnibus.